src/styles/blocks/page.css

```diff
@@ -0,0 +1,16 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
```

Comment:
let's import fonts via `index.html`, they seems not to work is importing via stylesheets (that's why tests failed)